lighting_advanced_search/wizard/product_advanced_search.py

```diff
@@ -141,7 +137,10 @@ def advanced_search(self):
                                     self.color_temperature_in_ids, 'or')
 
         # luminous flux
-        domain += prepare_in_domain(['source_ids.line_ids.color_temperature_flux_ids.flux_id'], self.flux_in_ids, 'or')
+        if self.flux_from_in:
+            if self.flux_to_in:
+                domain += [('source_ids.line_ids.color_temperature_flux_ids.flux', '>=', self.flux_from_in),
+                           ('source_ids.line_ids.color_temperature_flux_ids.flux', '<=', self.flux_to_in)]
```

Comment:
It should work different: If `from_in` is not null and `to_in` is null then we search everything greater or equal than `from_in`, and the same logic for the other cases. The best way to implement that is building a domain appending each clause in case `from_in` and/or `to_in` are defined. Don't follow the `wattage` example here.